src/impact-dashboard/ImpactDashboard.tsx

```diff
@@ -54,21 +54,7 @@ function useModel() {
     dispatch({ type: "update", payload: update });
   }
 
-  function resetModel(stateCode?: string, countyName?: string) {
```

Comment:
I don't understand at first glance why `resetModel` is no longer required?

Reply:
the logic of when to use it was too convoluted and distributed too much between here and the UI ... basically the work it did was factored out into two new functions, `getResetBase` and `getLocaleData`, that can be used independently of one another to achieve the same result. now the `update` action decides when it wants to reset the state, rather than having the input UI make that decision. (we don't want the input UI to have to know about the URL business, is how this all came about)

Reply:
Nice separation of concerns. Makes sense.